Tests/SunriseSet+HelperTests.swift

```diff
@@ -8,7 +8,7 @@ class Suntimes_HelperTests: XCTestCase {
     func testDaylightLengthProgress() {
         let coordinate = CLLocationCoordinate2D(latitude: 59.917, longitude: 10.75)
 
-        let timeZone = TimeZone(identifier: "UTC")!
+        let timeZone = TimeZone(secondsFromGMT: 0)!
```

Comment:
is this necessary because circle is running on GMT instead of UTC?

Reply:
I tried using UTC, and it returns GMT. 🤦‍♀️

Reply:
So looking at this [in a playground](https://gist.github.com/designatednerd/25f90de58ee3d492f2b55f209a0a886b), despite `UTC` showing up in `TimeZone`'s `abbreviationDictionary` as mapping from `"UTC"` to `"UTC"`, doing any of the following: 

```swift
let utc = TimeZone(identifier: "UTC")
let gmt = TimeZone(secondsFromGMT: 0)
let gmt2 = TimeZone(identifier: "GMT")
```

Prints out `GMT (fixed)` when logged. 

A clue seems to be that while `GMT` is in the list of `knownTimeZoneIdentifiers`, while `UTC` is not. But wow, that's confusing.

Reply:
Right? Aren't time zones complicated enough?!